catalog/transformer/catalog-transformer-mgmp/src/main/java/org/codice/alliance/catalog/transformer/mgmp/MgmpTransformer.java

```diff
@@ -0,0 +1,604 @@
+            throws IOException, CatalogTransformerException {
+        return super.transform(inputStream);
+    }
+
+    @Override
+    public Metacard transform(InputStream inputStream, String id)
+            throws IOException, CatalogTransformerException {
+        return super.transform(inputStream, id);
+    }
+
+    @Override
+    public LinkedHashSet<Path> buildPaths() {
+        LinkedHashSet<Path> paths = new LinkedHashSet<>();
+
+        Arrays.asList(MgmpConstants.RESOURCE_ORIGINATOR_SECURITY_PATH,
+                MgmpConstants.RESOURCE_SECURITY_RELEASABILITY_PATH,
+                MgmpConstants.LANGUAGE_PATH,
+                MgmpConstants.CLOUD_COVERAGE_PATH,
+                MgmpConstants.FORMAT_PATH,
+                GmdConstants.FORMAT_VERSION_PATH,
+
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_NUMBER_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_DATUM_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_DATUM_NUMBER_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_ELLIPSOID_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_ELLIPSOID_NUMBER_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_PROJECTION_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_PROJECTION_NUMBER_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_GRID_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_GRID_NUMBER_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_GRID_WKT_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_PROJECTION_WKT_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_ELLIPSOID_WKT_TYPE_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_DATUM_WKT_TYPE_PATH,
+
+                MgmpConstants.ISR_COVERAGE_CATEGORY_PATH,
+                MgmpConstants.ISR_COVERAGE_COMMENT_PATH,
+                MgmpConstants.ISR_IMAGE_COMMENT_PATH,
+                MgmpConstants.ISR_IMAGE_DESCRIPTION_PATH,
+                MgmpConstants.ISR_MD_IMAGE_COMMENT_PATH,
+                MgmpConstants.ISR_MD_IMAGE_DESCRIPTION_PATH,
+                MgmpConstants.ISR_VIDEO_COMMENT_PATH,
+                MgmpConstants.ISR_VIDEO_DESCRIPTION_PATH,
+
+                MgmpConstants.METADATA_ORIGINATOR_SECURITY_PATH,
+                MgmpConstants.METADATA_RELEASABILITY_PATH,
+                MgmpConstants.RESOURCE_SECURITY_RELEASABILITY_PATH,
+                MgmpConstants.NIIRS_PATH,
+                MgmpConstants.NIIRS_RATING_PATH,
+                MgmpConstants.RESOURCE_SECURITY_PATH,
+                MgmpConstants.METADATA_SECURITY_PATH,
+                MgmpConstants.MGMP_SPATIAL_REFERENCE_SYSTEM_CRS_WKT_TYPE_PATH)
+                .stream()
+                .map(this::toPath)
+                .forEach(paths::add);
```

Comment:
Instead of defining an empty map above and relying on side effect here, change this `forEach` to 
`.collect(Collectors.toCollection(LinkedHashSet::new))`